app/src/main/java/org/gdg/frisbee/android/onboarding/AppInviteLinkGenerator.java

```diff
@@ -63,4 +78,30 @@ private String createDeepLink(String gplusId) {
             .toString();
     }
 
+    private void shareShortAppInviteLink(String longUrl) {
+        FirebaseDynamicLinksHub firebaseDynamicLinksHub = App.from(activity).getFirebaseDynamicLinksHub();
+        String requestString = "{\"longDynamicLink\": \"" + longUrl + "\",\"suffix\": {\"option\": \"SHORT\"}}";
```

Comment:
Just like the response, can you create a `Request` class with these fields? 
Then Retrofit and Gson will automatically create the json body for you. This will be much cleaner.